### PR TITLE
ManualTicket element support for PNR_AddMultipleElements

### PR DESCRIPTION
**Pull request by bimusiek**
😎
Tested on PDT, ready to be merged.

### CODE REVIEW

File: tests/Amadeus/Client/Struct/Pnr/AddMultiElementsTest.php [hunk 1]
```diff
@@ -52,6 +53,8 @@
 use Amadeus\Client\Struct\Pnr\AddMultiElements\PnrActions;
 use Test\Amadeus\BaseTestCase;
 
+class UnsupportedElement extends Element { }
```
Comment (DerMika) [inline tests/Amadeus/Client/Struct/Pnr/AddMultiElementsTest.php]: Haha, I like this 😀 

I actually had to find some kind of not-yet-implemented element for this test, but I guess this is more future proof 😉

File: src/Amadeus/Client/Struct/Pnr/AddMultiElements/ElementManagementData.php [hunk 2]
```diff
@@ -132,6 +132,13 @@ class ElementManagementData
      */
     const SEGNAME_SEAT_REQUEST = "STR";
 
+    /**
+     * FHE - Manual Document Registration element - with Eletronic Ticket number
```
Comment (DerMika) [inline src/Amadeus/Client/Struct/Pnr/AddMultiElements/ElementManagementData.php]: Small typo in 'Electronic' here?

**Comment by DerMika** [comment]
You are on a roll lately!

I'll merge when you fix the style issues. And perhaps that typo I mentioned above.

**Comment by bimusiek** [comment]
@DerMika I think this was the last one for ATC 😅 I just finished all test cases for ATC certification and it looks like everything worked as expected. Let's see if Amadeus will approve all of the traces.

**Comment by DerMika** [comment]
Well I'm happy with all the contributions ;)

Hopefully the certification will go well!

**Pull request merged**